### PR TITLE
Linkify urls and formatting

### LINKED ISSUE #62: Confirm all URLs are links
MkDocs is much more strict with Markdown rendering so it would be good to confirm all URLs are linked as intended. I've seen a couple that are not and corrected, but assume there are others

### PR DESCRIPTION
**Pull request by dcwalk**
Fixes #62

### CODE REVIEW

**Review by titaniumbones — APPROVED** [review]
looks good!

File: docs/harvesting.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Your method for doing this will depend on the shape and size of the data you're
...
 
-The tricky part of this approach is generating a list of URLs to download from the page. If you're skilled with using scripts in combination with html-parsers (for example python's wonderful [beautiful-soup package](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start)), go for it. Otherwise, we've included the [jquery-URL-extraction guide](thttps://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-URL-extraction)], which has the advantage of working within a browser and can operate on a page that has been modified by JavaScript.
+The tricky part of this approach is generating a list of URLs to download from the page. If you're skilled with using scripts in combination with html-parsers (for example python's wonderful [beautiful-soup package](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start)), go for it. Otherwise, we've included the [jquery-URL-extraction guide](thttps://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-URL-extraction), which has the advantage of working within a browser and can operate on a page that has been modified by JavaScript.
```
Comment (suchthis) [inline docs/harvesting.md]: Looks like that jQuery URL needs to be updated from:
thttps://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-URL-extraction
to:
https://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-url-extraction

(and below for case-sensitivity)

File: docs/bagging.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ Baggers do some quality assurance on the dataset to make sure the content is cor
 - The organizers of the event (in-person or remote) will send you an invite to the [Archivers app](http://www.archivers.space/), which helps us coordinate all the data archiving work we do.
   	- Click the invite link, and choose a user name and a password.
 - Create an account on the DataRefuge Slack using this [slack-in](https://rauchg-slackin-qonsfhhvxs.now.sh/) (or use the Slack team recommended by your event organizers). This is where people share expertise and answer each other's questions.   
-- Get set up with Python and the Python script to make a bag at the command line: [https://github.com/LibraryOfCongress/bagit-python](https://github.com/LibraryOfCongress/bagit-python)
+- Get set up with Python and the [`bagit-python`](https://github.com/LibraryOfCongress/bagit-python) script to make a bag at the command line
 - If you need any assistance:
      - Talk to your DataRescue guide if you are at an in-person event.
-     - Or post questions on Slack in the #Baggers channel (or other channel recommended by your event organizers).
+     - Or post questions on Slack in the `#baggers` channel (or other channel recommended by your event organizers).
```
Comment (suchthis) [inline docs/bagging.md]: Should we update all channel names to lowercase? I'd just made them capped throughout for consistency, but they may not match actual channel names, depending on which Slack team is used by an event.

**Review by suchthis — APPROVED** [review]

File: docs/bagging.md (hunk 2 above)
Comment (dcwalk) [reply docs/bagging.md]: Hmmm, good question. I was just updating to the DataRefuge Slack name. I think we are okay with that, but maybe we specify "DataRefuge `#baggers` channel"?

File: docs/harvesting.md (hunk 1 above)
Comment (dcwalk) [reply docs/harvesting.md]: Thanks for the catch!

**Comment by dcwalk** [comment]
Merging in, given size of change and time before events

**Pull request merged**